test_helpers/fixtures/all_the_features.org

```diff
@@ -102,7 +105,7 @@ qui dolorem eum fugiat quo voluptas nulla pariatur?"
    CLOCK: [2019-11-11 Mon 13:12]--[2019-11-12 Tue 13:12] => 24:00
    CLOCK: [2019-10-13 Sun 12:15]--[2019-10-13 Sun 14:12] =>  1:57
    :END:
-** A messy headline with trailing whitespace                    
+** A messy headline with trailing whitespace
```

Comment:
Ups, did you remove the trailing whitespace here?

Reply:
Nope, Emacs did(;

Thanks for the quick review. I'll revert this of course.

Haven't read through the diff, yet. Only fixed the issue at hand. I would have hoped to also catch it, but you were blazingly fast. Thanks a bunch :pray:

Reply:
Haha, I'm faster than all tests :D

Reply:
Really impressive! :rocket: :godmode: